docs/quickstart/index.md

```diff
@@ -0,0 +1,152 @@
+
+??? info "Verify system compatibility"
+    - **Disk**: Approximately 1 GB of storage per monitored database node with data retention set to one week. By default, retention is 30 days.
+    - **Memory**: A minimum of 2 GB per monitored database node. The increase in memory usage is not proportional to the number of nodes. For example, the data from 20 nodes should be easily handled with 16 GB.
+    - **Ports**: Your system’s firewall should allow TCP traffic on port 443.
+
+## Install PMM
+
+The Easy-install script only runs on Linux-compatible systems. To use it, run the command with `sudo` privileges or as `root`:
+{ .power-number }
+
+1. Download and install PMM using `cURL` or `wget`:
+
+    === "cURL"
+
+        ```sh
+        curl -fsSL https://www.percona.com/get/pmm | /bin/bash
+        ```
+
+    === "wget"
+
+        ```sh
+        wget -qO - https://www.percona.com/get/pmm | /bin/bash    
+        ```
+
+2. After the installation is complete, log into PMM with the default `admin:admin` credentials.
+
+??? info "What's happening under the hood?"
+     This script does the following:
+
+     * Installs Docker if it is not installed on your system.
+     * Stops and renames any currently running PMM Docker container from `pmm-server` to `pmm-server-{timestamp}`. This old `pmm-server container is not a recoverable backup.
+     * Pulls and runs the latest PMM Docker image.
+
+### Connect database
+
+Once PMM is set up, choose the database that you want it to monitor:
+
+=== "MySQL"
+
+    Follow the instructions below to connect a Self-hosted MySQL database. Alternatively, you can connect an [AWS RDS](../setting-up/client/aws.md), [Azure MySQL](../setting-up/client/azure.md) or a [Google Cloud MySQL ](../setting-up/client/google.md) database.
+    { .power-number }
+
+    1. Create database account for PMM using the following command example. This creates a database user with name `pmm``, password `pass``, and the necessary permissions:
+
+        ```sql
+        CREATE USER 'pmm'@'127.0.0.1' IDENTIFIED BY 'pass' WITH MAX_USER_CONNECTIONS 10;
+        GRANT SELECT, PROCESS, REPLICATION CLIENT, RELOAD, BACKUP_ADMIN ON *.* TO 'pmm'@'127.0.0.1';
+        ```
+
+    2. To optimize server-side resources, install PMM Client via Package Manager on the database node:
+         
+        1. Install Percona Release Tool:
+
+            ```sh
+            wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+            dpkg -i percona-release_latest.generic_all.deb
+            ```
+        2. Install the PMM Client package:
+            
+            ```sh
+            apt update
+            apt install -y pmm2-client
+            ```
+
+        3. [Register database node with PMM](../setting-up/client/index.html#register):
+                
+            ```sh
+            pmm-admin config --server-insecure-tls --server-url=https://admin:admin@X.X.X.X:443
+            ```
```

Comment:
these 3 steps are identical for all three installments

Reply:
This file has been removed from this branch and will be added in main as its not related to PMM 2.41.0.